components/jwt-init/get-vault-secrets.sh

```diff
@@ -0,0 +1,227 @@
+#!/bin/bash
```

Comment:
is this a copy of a file? I think if it is then we should try and somehow have a check for consistency... or try to make it such that they point to the same file..